docs/docs/deployment.md

```diff
@@ -2,35 +2,26 @@
 
 ## Foreword
 
-As of beginning 2016 the docker toolset is not quite there to provide a heroku like experience
-when deploying to production. A lot of parts are already there, but have a few quirks that need
-to be addressed.
-
-Because of that, the deployment strategy for djangopackages.org is a bit different from what you
-read in the getting started with docker tutorials.
-
-First, we don't use docker-machine. It's not reliable and has no team of maintainers comparable
-to other distros like debian, ubuntu or rhel that manages security releases.
-
-Second, there's no way do daemonize the docker compose process. When the underlying VM is
-restarted, the stack won't start automatically.
+djangopackages.org use Docker Compose both for local development and in production.
 
 The current strategy is:
 
-> - Use a virtual machine with a well patched OS (debian, ubuntu, RHEL), djangopackages.org is using
->   ubuntu 14.04
-> - Install docker, docker-compose, git and supervisord
-> - Clone the code on the server
-> - Let supervisord run it
+- Use a virtual machine with a well patched OS (debian or ubuntu), djangopackages.org is using
+- Install Docker, Docker Compose, git, and supervisord (manages Docker)
+- Clone the code on the server
+- Start our services
 
 ## Stack
 
-The configuration in `docker-compose.yml` contains 4 services:
+All of our `docker-compose*.yml` configurations contains the following services:
 
-> - `postgres` that powers the database
-> - `django-a` and `django-b` that runs the WSGI server and serves the app through gunicorn
-> - `caddy` that proxies incoming requests to the gunicorn server
-> - `redis` as cache
+- `postgres` powers our database.
```

Comment:
Maybe we can mention that we are using managed PostgreSQL in production?

Reply:
Great idea. 

Side-note: I'd like to remove the `postgres/Dockerfile` at some too since that feels like part of a different era and I don't see where that's useful anymore.

Reply:
Sounds good, created an issue to track this https://github.com/djangopackages/djangopackages/issues/1077